tools/python/google_maps_bookmarks.py

```diff
@@ -2,97 +2,133 @@
 
 import csv
 import json
-import os.path
+import argparse
+import mimetypes
 import traceback
 import urllib.error
 import urllib.parse
 import urllib.request
 import xml.etree.ElementTree as ET
-from os import R_OK
+from os import path, access, R_OK, linesep
+from io import StringIO
 
 
-class GoogleMapsConverter:
+from dateutil import parser as dateutil_parser
```

Comment:
It looks like `dateutil` is not part of the python distribution and the user has to install this module via `pip`. The script should run without any external dependencies. Could you replace `dateutil` with `datetime`?